UPGRADING.md

```diff
@@ -7,6 +7,12 @@ Please make sure to create a MongoDB database backup before starting the upgrade
 
 ## Breaking Changes
 
+### Breaking API changes ##
+
+| Endpoint                        | Description                 |
+| ------------------------------- | --------------------------- |
+| `GET /api/system/configuration` | The key `stale_master_timeout` has been been replaced with `stale_leader_timeout` in the response object. |
```

Comment:
I thought was OK to make it a breaking change, but was not really sure myself. See PR description. To avoid any potential issues I made it non-breaking as per your suggestion. I don't know of any good way to properly deprecate a field in a response and announce it to clients though. I guess usually you'd switch to a new API version at some point and deprecate the whole endpoint.